specs/slider/Spec.md

```diff
@@ -1,199 +1,214 @@
+- [API](#api)
 
-Sliders are great for adjusting settings that reflect intensity levels (volume, brightness, color saturation).
+- [States](#states)
 
+- [Accessibility](#accessibility)
 
+- [Behavior](#behavior)
 
-## Elements
+  - [Input Methods](#input-methods)
+    - [Keyboard](#keyboard)
+    - [Mouse](#mouse)
+    - [Touch](#touch)
 
-![elements](./assets/elements.png)
+- [RTL](#rtl)
 
-**Slider consists of:** "bar", "handle" and "progress". The "handle" is dragged across the "bar" in order to give the slider a desired value while "progress" marks range from min value to current value. 
+- [DOM structure](#dom-structure)
 
-Also, you can display tooltip (e.g. to display current value) by adding child DOM element with relevant data-slot (data-slot="tooltip")
+- [Design](#design)
 
-## Visual States
+  ​
 
-| State    | Description                              |
-| :------- | ---------------------------------------- |
-| Default  | Default component appearance             |
-| Hover    | User hovered over bar OR handle          |
-| Focus    | Browser is focused on the component      |
-| Active   | User clicks on bar OR handle             |
-| Disabled | Component can not be changed             |
-| Error    | Error state for the component (can be set with :error pseudo-class) |
 
-Design [assets](https://zpl.io/2kRTvO)
 
+## Definition
 
+A **slider** an input where the user selects a value from within a given range.
 
-## Props
+Sliders are great for adjusting settings that reflect intensity levels (volume, brightness, color saturation).
 
-See [README.md](./README.md) for more info.
 
 
+## Elements
 
-## UI Customizations
+![elements](./assets/elements.png)
 
-Slider can be customized using ::handle, ::bar and ::progress subcomponents.
+**Slider consists of:** ::bar, ::handle, ::progressBar, ::marks and ::progressMarks . The ::handle is dragged across the bar in order to give the slider a desired value. ::progressBar shows range from min value to the current value. ::marks and ::progressMarks are representing the step
 
-See [README.md](./README.md) for more info.
+> TBD (need to decide on how to implement tooltip)
+> Also, you can display tooltip (e.g. to display current value) by adding child DOM element with relevant data-slot (data-slot="tooltip")
 
+## API
 
+**Props**
 
-## Behavior
+See [README.md](https://github.com/wix/stylable-components/blob/master/specs/specTemplate/README.md) for more info.
 
-User can drag handle over slider bar or click on the slider (in desired location) to select the value from the range.
+**Style**
 
-Changing the value (keyboard) is performed **from current value** to the next expected value. 
-E.g. if min=0, max=10, step=2, value=3.5, then UP arrow key will give us 4 and Down arrow key will give us 2
-Value can not exceed the min/max limits. If value is > or < than min/max it is automatically set to corresponding min/max.
+Slider can be customized using ::handle, ::bar, and ::progress subcomponents.
```

Comment:
This sentence should be updated according to latest changes (progressBar, marks, etc)